packages/model/tsconfig.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
It might be nice to have a default ts config file that we use to generate/copy these from

Reply:
Yes, that was part of the work I started to do with the default blueprints for packages, it would also let us "re-init" packages to bring them up to speed as a syncing mechanism